circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -705,6 +748,12 @@ class CircularSeekBar @JvmOverloads constructor(
         pointerPaint.strokeJoin = Paint.Join.ROUND
         pointerPaint.strokeCap = circleStyle
 
+        pointerOverlayPaint.set(pointerPaint)
+        pointerOverlayPaint.color = pointerOverlayColor
+        pointerOverlayPaint.style = Paint.Style.STROKE
+        pointerOverlayPaint.strokeWidth =
+            (pointerStrokeWidth - pointerOverlayBorderWidth).coerceAtLeast(0f)
```

Comment:
If we use the concept of "overlay", than the border width of this overlay looks more like it's own width (circle diameter).

So at here we may directly set the `pointerOverlayPaint.strokeWidth = pointerOverlayBorderWidth`